### ISSUE TITLE
Importing Bank of Cyprus OFX version 2.00?

### ISSUE THREAD

**Issue by whotopia** [OCA/bank-statement-import#68]
Having trouble importing OFX from my Bank of Cyprus statements.

Does the module support only version 1.0 format?  

Thanks,

**Comment by pedrobaeza** [comment]
Check if ofxparse library supports it in first place and which version you have.

**Comment by whotopia** [comment]
It seems like underlying ofxparse does.  There is file as part of unit tests ofxparse/tests/fixtures/multiple_accounts.ofx    which is essentially the same as format I am using.   

Error I'm seeing upon import is:
Odoo Warning
Could not make sense of the given file.
Did you install the module to support this type of file?

There is nothing in the log files.  

Any way to debug?   Does bank-statement-import just recognise files by extension? Is there any other config?  

I also have modules installed:
account_bank_statement_import_usability
account_import_line_multicurrency_extension

The relevant the bank account in Odoo is named according to it's IBAN, but ofx only has the account number.  Would that matter?

Thanks for quick response!

**Comment by pedrobaeza** [comment]
Odoo doesn't recognise by extension, but with a pattern check. Please inspect code to see the way it's handled.

**Comment by whotopia** [comment]
I turned on full debuggeing. Upon import, the logs shows  
![screen shot 2016-06-10 at 10 52 14](https://cloud.githubusercontent.com/assets/9039050/15959369/bd616fba-2efa-11e6-8306-879220bd5ee0.png)

I have suspicion that it never gets to the ofxparse module.  Any more suggestions appreciated.
Thanks
S

**Comment by pedrobaeza** [comment]
Sorry, no more clues. But you should inspect code, because these modules don't output anything in the log.

**Comment by whotopia** [comment]
Looking at the Chrome logs it appears the .ofx file doesn't even get uploaded to the backend.  Same problem with .qif files (I tried that plugin too).    Not sure what's going on here as I upload files all the time.

**Comment by whotopia** [comment]
I tried this with a completely clean installation of odoo 8 nightly build.  Same issue.

**Comment by whotopia** [comment]
SOLVED:    The OFX files output by Bank of Cyprus have some lines in their files terminated with /r/n. 
In my case to successfully import I had to run:
`dos2unix filename.ofx
`
Then I also needed to adjust the line:
`<ACCTID>nnnnnnnnnnnnn</ACCTID>`
to contain exactly the same account name as I specified in Odoo.  In my case this was IBAN formatted account name. 

After these 2 operations, everything seemed to work just fine.
